app/controllers/openid_application_controller.rb

```diff
@@ -198,6 +215,27 @@ def fetch_smart_launch_context
     raise error_klass('Invalid launch context')
   end
 
+  def fetch_issued(token_string)
+    return nil unless Settings.oidc.issued_url
+
+    response = RestClient.get(Settings.oidc.issued_url,
+                              { Authorization: 'Bearer ' + token_string })
+    raise error_klass('Invalid token') if response.nil?
+
+    if response.code == 200
+      json_response = JSON.parse(response.body)
+      if json_response['scopes']
+        json_response['scp'] = json_response['scopes']
+        json_response['scopes'] = nil
+      end
+      json_response
+    end
+  rescue => e
+    raise raise error_klass('Invalid token') if e.to_s.include?('Unauthorized')
```

Comment:
What conditions cause this rescue block to execute? Any non 200 response?

Reply:
This would be if the call succeeds to /issued endpoint, but there isn't a corresponding entry for the access token.

And then the raise below is if the call itself fails for any other reason, like misconfigured endpoint.